index.js

```diff
@@ -19,14 +19,19 @@ const baseLink = "https://www.icloud.com/shortcuts/";
  * A shortcut metadata.
  */
 class ShortcutMetadata {
-	constructor(object) {
-		// Not sure if we keep the native Apple names for the properties?
-		this.WFWorkflowClientRelease = object.WFWorkflowClientRelease;
-		this.WFWorkflowMinimumClientVersion = object.WFWorkflowMinimumClientVersion;
-		this.WFWorkflowIcon = object.WFWorkflowIcon;
-		this.WFWorkflowImportQuestions = object.WFWorkflowImportQuestions;
-		this.WFWorkflowTypes = object.WFWorkflowTypes;
-		this.WFWorkflowInputContentItemClasses = object.WFWorkflowInputContentItemClasses;
+	constructor(metadata) {
+		this.client = {
+			minimumVersion: metadata.WFWorkflowMinimumClientVersion,
+			version: metadata.WFWorkflowClientRelease,
+		};
+		this.icon = {
```

Comment:
Can these properties be named similarly to the ones in a `Shortcut`, such as `color`, `data`, and `glyph`?